laboratory-servlet-api-integration/src/it/scala/com/wixpress/petri/test/LaboratoryPropertiesIT.scala

```diff
@@ -0,0 +1,23 @@
+package com.wixpress.petri.test
+
```

Comment:
any reason this file was added to laboratory-servlet-api-integration instead of to the petri-test-utils module?